### DISCUSSION TITLE
modular home path

### DISCUSSION THREAD

**Discussion by BlackHo0d** [modular/modular#1201] [Q&A]
Hi, I installed mojo and now it's required to set "Mojo: Modular Home Path" "The path to the MODULAR_HOME containing the Mojo SDK installation." 
Only one line is required and it's not export MODULAR_HOME="$HOME/.modular"
export PATH="$MODULAR_HOME/pkg/packages.modular.com_mojo/bin:$PATH"

I use Mac os(Intel)

**Comment by Sharktheone** [comment]
So, i guess you mean the Modular Home Path in VSCode. For me it just was `/home/shark/.modular`. I think, it doesn't like the usage of other environment variables in the path. I've tested it on Linux, when I use `$HOME/.modular` it doesn't work, but when using `/home/shark/.modular/`it is fine. I don't know, how `$HOME` looks on MacOS, but you just paste the output from `echo $HOME/.modular` into the modular home path.

**Comment by BlackHo0d** [comment]
I've pasted output from echo $HOME/.modular but it didn't work

I've got: Mojo SDK initialization error: The mojo.modularHomePath setting '/Users/rustam/.modular' does not exist.

**Comment by Sharktheone** [comment]
Hmm, maybe the `.modular` folder is somewhere else on MacOS, what is the folder shown in `which mojo`?

**Comment by BlackHo0d** [comment]
@MacBook-Pro-Rustam ~ % which mojo 
mojo not found
@MacBook-Pro-Rustam ~ %

**Comment by BlackHo0d** [comment]
I've installed mojo extensions, CLI and SDK but can not set the path to mojo. 
I got this :
The Mojo🔥 development environment was not found. If the Mojo SDK is installed, please set the MODULAR_HOME environment variable to the appropriate path, or set the `mojo.modularHomePath` configuration. If you do not have it installed, would you like to install it?

Mojo SDK is installed. I've installed all required extensions.

**Comment by Sharktheone** [comment]
hmm, and what does `which modular`? You probably didn't add mojo to your $PATH, but this should be okay when using only the VSCode extension for running

**Comment by BlackHo0d** [comment]
I got: modular not found

How to add mojo to my $PATH?

**Comment by Sharktheone** [comment]
Okay, how did you then install mojo? I mean you need modular to install mojo.

**Comment by BlackHo0d** [comment]
I've done all steps which was provided on Modular website. I installed Docker, mojo and docker extensions, CLI, SDK, I wrote first program in repl but I can not run mojo files. It's required to set a path, but everything I did didn't work(including your advices)

**Comment by Sharktheone** [comment]
Ohh, you have installed it with docker. I've read somewhere, you need mojo running on your local device for the LSP and stuff. Please upvote #1092 for MacOS Intel support
